crates/bevy_asset/src/processor/mod.rs

```diff
@@ -18,7 +18,7 @@ use crate::{
 };
 use bevy_ecs::prelude::*;
 use bevy_log::{debug, error, trace, warn};
-use bevy_tasks::IoTaskPool;
+use bevy_tasks::ComputeTaskPool;
```

Comment:
If it is the only TaskPool remaining, would it make sense to name this simply TaskPool? 
Otherwise it sounds like there are non-compute TaskPools

Reply:
> I don't want to push that kind of breakage in this PR. The focus here is to get the functionality in place.

It was already answered in previous comment.